packages/edit-widgets/src/index.js

```diff
@@ -33,11 +33,7 @@ export function initialize( id, settings ) {
 		( block ) => ! [ 'core/more' ].includes( block.name )
 	);
 	registerCoreBlocks( coreBlocks );
-	if ( process.env.GUTENBERG_PHASE === 2 ) {
-		__experimentalRegisterExperimentalCoreBlocks( {
-			enableLegacyWidgetBlock: true,
-		} );
-	}
+	__experimentalRegisterExperimentalCoreBlocks();
```

Comment:
@noisysocks I think this should be behind the Gutenberg phase flag like it was before.

Reply:
I guess we only needed to remove the `enableLegacyWidgetBlock` option (and options object) being passed as an argument.

Reply:
Oh, I see #32138 now, ignore this!